vivisect/tests/testswitches.py

```diff
@@ -0,0 +1,187 @@
+import unittest
```

Comment:
I like that we now have tests for this, but also there should be some switchcases in the binaries that we already analyze as part of the tests, so I'm curious to see how this performs in the context of a full binary.